chef/cookbooks/crowbar-hacks/recipes/default.rb

```diff
@@ -43,7 +43,7 @@
       mode "0644"
       variables(logfiles: "/var/log/crowbar/*.log /var/log/crowbar/*.out /var/log/crowbar/chef-client/*.log",
                   action: "create 644 crowbar crowbar",
-                postrotate: "/usr/bin/killall -USR1 rainbows")
+                postrotate: "/usr/bin/pumactl -S /opt/dell/crowbar_framework/tmp/pids/puma.state stop")
```

Comment:
I would suggest `phased-restart`, these are the available actions of pumactl: halt|restart|phased-restart|start|stats|status|stop|reload-worker-directory

@vuntz what's your opinion on that?